integration-tests/cli/csv-parser-price.it.js

```diff
@@ -111,10 +111,12 @@ describe('CSV and CLI Tests', () => {
       const csvFilePath = './packages/csv-parser-price/test/helpers/faulty-sample.csv'
 
       exec(`${binPath} -p ${projectKey} -i ${csvFilePath} --logLevel verbose`,
-        (error, stdout, stderr) => {
+        (error, stdout) => {
           expect(error.code).toBe(1)
-          expect(stdout).toBeFalsy()
-          expect(stderr).toMatch(/\.js:\d+:\d+/)
+          expect(stdout).toMatch('{"prices":[]}')
+
+          const errorLog = fs.readFileSync('csvparserprice.log', 'utf8')
```

Comment:
we can make the log file customizable and create a tmp file for this test, I feel

but then, if there is an error that makes the script to stop, we probably want to output that to the user *and* log it to a file...